src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+abstract class Figure implements IFigure {
```

Comment:
The first line after the class declaration should not be an empty line. Please remove the unnecessary empty line.